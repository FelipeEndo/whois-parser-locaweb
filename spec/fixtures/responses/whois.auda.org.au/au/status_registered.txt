Domain Name: GOOGLE.COM.AU
Registry Domain ID: D407400000001774763-AU
Registrar WHOIS Server: whois.auda.org.au
Registrar URL:
Last Modified: 2021-08-05T15:18:41Z
Registrar Name: MarkMonitor Corporate Services Inc
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller Name:
Status: clientDeleteProhibited https://afilias.com.au/get-au/whois-status-codes#clientDeleteProhibited
Status: clientUpdateProhibited https://afilias.com.au/get-au/whois-status-codes#clientUpdateProhibited
Status: serverDeleteProhibited https://afilias.com.au/get-au/whois-status-codes#serverDeleteProhibited
Status Reason: Registry Lock
Status: serverRenewProhibited https://afilias.com.au/get-au/whois-status-codes#serverRenewProhibited
Status Reason: Not Currently Eligible For Renewal
Status: serverTransferProhibited https://afilias.com.au/get-au/whois-status-codes#serverTransferProhibited
Status Reason: Registry Lock
Status: serverUpdateProhibited https://afilias.com.au/get-au/whois-status-codes#serverUpdateProhibited
Status Reason: Registry Lock
Registrant Contact ID: MMR-174095
Registrant Contact Name: Domain Admin
Tech Contact ID: MMR-171195
Tech Contact Name: Domain Administrator
Name Server: NS1.GOOGLE.COM
Name Server: NS2.GOOGLE.COM
Name Server: NS3.GOOGLE.COM
Name Server: NS4.GOOGLE.COM
DNSSEC: unsigned
Registrant: Google LLC
Eligibility Type: Trademark Owner
Eligibility Name: GOOGLE
Eligibility ID: TM 788234

>>> Last update of WHOIS database: 2021-12-14T11:04:43Z <<<



Afilias Australia Pty Ltd (Afilias), for itself and on behalf of .au Domain Administration Limited (auDA), makes the WHOIS registration data directory service (WHOIS Service) available solely for the purposes of:

(a) querying the availability of a domain name licence;

(b) identifying the holder of a domain name licence; and/or

(c) contacting the holder of a domain name licence in relation to that domain name and its use.

The WHOIS Service must not be used for any other purpose (even if that purpose is lawful), including:

(a) aggregating, collecting or compiling information from the WHOIS database, whether for personal or commercial purposes;

(b) enabling the sending of unsolicited electronic communications; and / or

(c) enabling high volume, automated, electronic processes that send queries or data to the systems of Afilias, any registrar, any domain name licence holder, or auDA.

The WHOIS Service is provided for information purposes only. By using the WHOIS Service, you agree to be bound by these terms and conditions. The WHOIS Service is operated in accordance with the auDA WHOIS Policy (available at https://www.auda.org.au/policies/index-of-published-policies/2014/2014-07/ ).
