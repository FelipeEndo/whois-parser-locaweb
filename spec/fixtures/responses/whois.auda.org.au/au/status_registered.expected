#disclaimer
  %s == "Afilias Australia Pty Ltd (Afilias), for itself and on behalf of .au Domain Administration Limited (auDA), makes the WHOIS registration data directory service (WHOIS Service) available solely for the purposes of:\n(a) querying the availability of a domain name licence;\n(b) identifying the holder of a domain name licence; and/or\n(c) contacting the holder of a domain name licence in relation to that domain name and its use.\nThe WHOIS Service must not be used for any other purpose (even if that purpose is lawful), including:\n(a) aggregating, collecting or compiling information from the WHOIS database, whether for personal or commercial purposes;\n(b) enabling the sending of unsolicited electronic communications; and / or\n(c) enabling high volume, automated, electronic processes that send queries or data to the systems of Afilias, any registrar, any domain name licence holder, or auDA.\nThe WHOIS Service is provided for information purposes only. By using the WHOIS Service, you agree to be bound by these terms and conditions. The WHOIS Service is operated in accordance with the auDA WHOIS Policy (available at https://www.auda.org.au/policies/index-of-published-policies/2014/2014-07/ )."


#domain
  %s == "GOOGLE.COM.AU"

#domain_id
  %s == "D407400000001774763-AU"


#status
  %s == ["clientDeleteProhibited https://afilias.com.au/get-au/whois-status-codes#clientDeleteProhibited", "clientUpdateProhibited https://afilias.com.au/get-au/whois-status-codes#clientUpdateProhibited", "serverDeleteProhibited https://afilias.com.au/get-au/whois-status-codes#serverDeleteProhibited", "serverRenewProhibited https://afilias.com.au/get-au/whois-status-codes#serverRenewProhibited", "serverTransferProhibited https://afilias.com.au/get-au/whois-status-codes#serverTransferProhibited", "serverUpdateProhibited https://afilias.com.au/get-au/whois-status-codes#serverUpdateProhibited"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %CLASS{time}
  %s %TIME{2021-08-05 15:18:41 UTC}

#expires_on
  %s %ERROR{AttributeNotSupported}


#registrar
  %s %CLASS{registrar}
  %s.name         == "MarkMonitor Corporate Services Inc"
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "MMR-174095"
  %s[0].name          == "Domain Admin"
  %s[0].organization  == "Google LLC"
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "MMR-171195"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == nil
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "NS1.GOOGLE.COM"
  %s[1] %CLASS{nameserver}
  %s[1].name == "NS2.GOOGLE.COM"
  %s[2] %CLASS{nameserver}
  %s[2].name == "NS3.GOOGLE.COM"
  %s[3] %CLASS{nameserver}
  %s[3].name == "NS4.GOOGLE.COM"
